lib/screens/details_page/details.dart

```diff
@@ -32,267 +30,315 @@ class Details extends StatefulWidget {
-                              onPressed: () {
-                                Navigator.pushNamed(context, Routes.scan);
-                              },
+                            Text(
+                              'Description',
+                              style: TextStyle(
+                                  fontWeight: FontWeight.bold, fontSize: 20),
                             ),
-                            const SizedBox(
-                              width: 200,
-                            ),
-                            //price
                           ],
                         ),
+                      )),
+                      const VerticalDivider(
+                        width: 1,
+                      ),
+                      Expanded(
+                          child: const Center(
+                        child: Text(
+                          ' Nutritional value',
+                          style: TextStyle(
+                              fontWeight: FontWeight.bold, fontSize: 15),
+                        ),
+                      )),
+                    ],
+                  ),
+                  const Divider(
+                    indent: 220,
+                    endIndent: 20,
+                    thickness: 1,
+                    color: Colors.black,
+                  ),
+                  Row(
+                    children: [
+                      Expanded(
+                        flex: 1,
+                        child: Container(
+                          padding: const EdgeInsets.only(
+                              left: 50, right: 10, top: 0, bottom: 40),
+                          child: Text(
+                            widget.description,
+                          ),
+                        ),
                       ),
-                      Row(
-                        mainAxisAlignment: MainAxisAlignment.center,
-                        children: [
-                          Expanded(
-                              child: Center(
+                      const VerticalDivider(
+                        width: 1.0,
+                      ),
+                      Expanded(
+                          child: Container(
+                        padding: const EdgeInsets.only(left: 30, right: 10),
+                        child: Center(
                             child: Column(
-                              children: [
-                                const Text(
-                                  'Description',
-                                  style: TextStyle(
-                                      fontWeight: FontWeight.bold,
-                                      fontSize: 20),
-                                ),
-                              ],
+                          children: [
+                            const Text(
+                              'omg cheese cheese cheese why cheese ahhhhhhhhh cheeseeee echeeeeeeeessseeeeee ',
```

Comment:
@batsy3 same applies to this, i expect to see variable names representing values coming from the api and not static dummy text.

Reply:
i need to add some other models in the backend for these things and a few others